gallery/file_modules/__init__.py

```diff
@@ -39,6 +39,17 @@ def generate_thumbnail(self):
         self.thumbnail_uuid = hash_file(self.file_path)
 
         with Image(filename=self.file_path) as img:
+
+			# read metadata to find orientation of image
+			# then rotate accordingly. 
+			rotation = img.metadata["exif:Orientation"]
```

Comment:
Can you fix the indentation? Looks like you're maybe using tab characters which aren't playing nice w/ the surrounding 4-space tabs.

Reply:
In addition to this, where are you finding `metadata`? Looks like we use exif dicts, with a `get_exif()` function.

Reply:
From what I remember and just looked up, metadata is just a read-only part of Image that contains the exif data. This implementation is just what I found about how to get exif data and it worked so I didn't question it.

Reply:
Also ew 4 space tabs

Reply:
Please see [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation) for indentation.

Reply:
Can I easily edit change this without making a new PR? I cleaned it up locally

Reply:
Yeah just commit and push to your branch.